src/rprblender/nodes/blender_nodes.py

```diff
@@ -1264,9 +1264,17 @@ def export(self):
             elif op == 'SCALE':
                 # input 2 here is a scalar
                 res = in1 * in2
-            else:
-                raise ValueError("Incorrect operation", op)
+            else:  # 3-inputs operations
+                in3 = self.get_input_value(2)
 
+                if op == 'WRAP':
+                    val_range = in2 - in3
+                    if range != 0.0:
+                        res = in1 - val_range * ((in1 - in3) / val_range).floor()
```

Comment:
where did you find such formula?

Reply:
Cycles

Reply:
Please note so in a comment.  This will explain better for future viewers of code.